pages/common/jhat.md

```diff
@@ -2,14 +2,14 @@
 
 > Java Heap Analysis Tool.
 
-- Analyze a heap dump (from jmap), view via http on port 7000:
+- Analyze a heap dump (from jmap), view via HTTP on port 7000:
 
 `jhat {{dump_file.bin}}`
 
 - Analyze a heap dump, specifying an alternate port for the http server:
 
 `jhat -p {{port}} {{dump_file.bin}}`
 
-- Analyze a dump letting jhat use up to 8GB RAM (2-4x dump size recommended):
+- Analyze a dump letting jhat use up to 8 GB RAM (2-4x dump size recommended):
 
 `jhat -J-mx8G {{dump_file.bin}}`
```

Comment:
```suggestion
`jhat -J-mx{{8G}} {{dump_file.bin}}`
```

Reply:
Looking at the docs, I think it'd be better to do this instead?

```
-J<flag>
             Pass <flag> to the Java virtual machine on which jhat is run. For
             example, -J-Xmx512m to use a maximum heap size of 512MB.
```

```suggestion
`jhat -J{{-mx8G}} {{dump_file.bin}}`
```

The whole `-mx8G` is the user-input, not just `8G`.
I think it accepts any Java VM argument, though.

Reply:
Yeah, you're definitely right about that, but the example itself was only about about changing the max memory.

Reply:
My understanding is that it's _all_ user-defined input that needs to be put between curly braces?

> Basically, everything which isn't a flag, argument or subcommand should be in {{ }}. Also it makes the page more readable.
> ![](https://user-images.githubusercontent.com/50295997/117638178-dc034a00-b182-11eb-94b3-863d055faa19.png)
> ![](https://user-images.githubusercontent.com/50295997/117638133-d279e200-b182-11eb-866c-1a322c48e19b.png)
> \- https://github.com/tldr-pages/tldr/pull/5921#issuecomment-836462426

In the example in the PR the command was also for a very specific action, however user-defined parameters should still be wrapped, if I understand correctly.

Reply:
It's mostly contextual, but I think you're right here.